src/formatters.jl

```diff
@@ -8,7 +8,7 @@ log `Record`.
 abstract type Formatter end
 
 const DEFAULT_FMT_STRING = "[{level} | {name}]: {msg}"
-const DATE_FMT_STRING = "yyyy-mm-dd HH:MM:SS"
+const DEFAULT_DATE_FMT_STRING = "yyyy-mm-dd HH:MM:SS"
```

Comment:
Feels a bit verbose. Maybe we could just make it `DEFAULT_DATE_FMT`?

Reply:
Sure! Done in adfe69f9f3e5edd848f6e7909bd06f616d68f1a3